app/config/firewalls.yml

```diff
@@ -49,6 +49,8 @@ security:
             #context: admin_context
             provider: adminuser
             pattern: ^/admin?.*$
+            simple-preauth:
```

Comment:
Isn't the correct yaml key [`simple_preauth`](http://symfony.com/doc/current/cookbook/security/api_key_authentication.html#configuration) ?